lms/resources/_js_config.py

```diff
@@ -185,6 +214,24 @@ def _auth_token(self):
             self._request.lti_user
         )
 
+    def _canvas_files_available(self):
+        """Return True if the Canvas Files API is available to this request."""
+
+        if not self._is_canvas():
```

Comment:
```suggestion
        if not self.context.is_canvas():
```

Reply:
Oh yeah, I merged a PR to move this method (https://github.com/hypothesis/lms/pull/1520) and then forgot about it. Good catch